content/en/docs/setup/cri.md

```diff
@@ -0,0 +1,169 @@
+---
+reviewers:
+- vincepri
+- bart0sh
+title: CRI installation
+content_template: templates/concept
+weight: 100
+---
+{{% capture overview %}}
+Since v1.6.0, Kubernetes has enabled the use of CRI, Container Runtime Interface, by default.
+This page contains installation instruction for various runtimes.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+### Docker
+
+On each of your machines, install Docker.
+Version 18.06 is recommended, but 1.11, 1.12 and 1.13 are known to work as well.
+Versions 18.06+ _might work_, but have not yet been tested and verified by the Kubernetes node team.
```

Comment:
like i pointed out, sig-node will not verify anything post 17.03 :\
they have not verified 18.06 too.

best to change this sentence to:
`Versions 18.06+ _might work_, but have not yet been tested and verified.`

Reply:
my bad. Missed your point here. Is it better now?